conf.yaml

```diff
@@ -870,6 +870,10 @@ contents:
               -
                 repo:   beats
                 path:   libbeat/docs
+              -
+                repo:   beats
+                path:   libbeat/processors/*/docs/*.asciidoc
```

Comment:
Could you drop the `*.asciidoc` bits here?

Nothing else has them and they'll be surprising if we ever want to include an image in these directories or something like that.